src/examples/notifications/global-alert/GlobalAlertTypes.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { GlobalAlert } from '@zendeskgarden/react-notifications';
+import { Anchor } from '@zendeskgarden/react-buttons';
+
+const StyledSpacer = styled.div`
+  height: ${p => p.theme.space.md};
+`;
+
+const Example = () => (
+  <>
+    <GlobalAlert type="info">
+      <GlobalAlert.Title>New update available</GlobalAlert.Title>
```

Comment:
This example is highlighting a styling fault with `GlobalAlert`. On first glance, I think there is a missing `flex-shrink: 0` on this element. Please note for a follow-on component fix.

Reply:
I'll create a PR to fix this.

Reply:
@Francois-Esquire Can you pls create a ticket for this? thx!

Reply:
@smritimadan for sure, I'll do that!

Reply:
Raised a PR in the meantime, [`fix(notifications): add flex-shrink to global alert title`)](https://github.com/zendeskgarden/react-components/pull/1464).

Reply:
Actually @Francois-Esquire, I think the structure is incorrect here. Based on late game component discussions, the `GlobalAlert.Title` is meant to live _within_ `GlobalAlert.Content`. I don't think there is a need to update the component. Rather, these examples should be restructured.